cy.json

```diff
@@ -0,0 +1,190 @@
+{
+    "name" : "Cymraeg",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
+            "dateFormat" : "[month] [date] [year]",
+            "months" : [
+                "Ionawr",
+                "Chwefror",
+                "Mawrth",
+                "Ebrill",
+                "Mai",
+                "Mehefin",
+                "Gorffennaf",
+                "Awst",
+                "Medi",
+                "Hydref",
+                "Tachwedd",
+                "Rhagfyr"
+            ],
+            "types" : {
+                "backpack" : "Bling Cefn",
+                "emote" : "Emote",
+                "glider" : "Gleider",
+                "emoji" : "Emoji",
+                "loading" : "Llwytho Sgrin",
+                "outfit" : "Gwisg",
+                "pickaxe" : "Pickaxe",
+                "skydive" : "Llwybr Plymio",
+                "umbrella" : "Ymbarél",
+                "spray" : "Chwistrell",
+                "toy" : "Tegan",
+                "pet" : "Anifail Anwes",
+                "music" : "Cerddoriaeth",
+                "wrap" : "Lapio",
+                "bundle" : "Bwndel",
+                "misc" : "Misc"
+            },
+            "rarity" : {
+                "legendary" : "Chwedlonol",
+                "epic" : "Epig",
+                "rare" : "Prin",
+                "uncommon" : "Anghyffredin",
+                "common" : "Cyffredin"
+            },
+            "translationNotAvailable" : "Unfortunately there is no translation available for this page.",
+            "translationIncomplete" : "Not all text on this page will be translated"
+        },
+        "nav" : {
+            "home" : "cartref",
+            "shop" : "siop",
+            "cosmetics" : {
+                "dropdown" : "colur",
+                "upcoming" : "sydd i ddod",
+                "list" : "rhestr",
+                "png" : "png",
+                "gallery" : "galeri",
+                "icons" : "eiconau",
+                "reminders" : "atgoffa",
+                "history" : "hanes y siop"
+            },
+            "news" : "newyddion",
+            "modes" : "dulliau",
+            "randomiser" : "ar hap",
+            "about" : {
+                "dropdown" : "am",
+                "about" : "am fnbr.co",
+                "privacy" : "polisi preifatrwydd",
+                "donate" : "Rhoddwch",
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "cyfrif",
+                "account" : "cyfrif",
+                "logout" : "allgofnodi"
+            },
+            "languagePicker" : {
+                "trigger" : "iaith",
+                "title" : "Dewis Iaith",
+                "currentLanguage" : "Ar hyn o bryd rydych chi'n defnyddio [iaith]."

```

Comment:
Can you change the text within [] back to [language] it is needed for the language to be filled in automatically